tests/core/crypto/test_crypto.h

```diff
@@ -68,6 +71,39 @@ TEST_CASE("[Crypto] PackedByteArray constant time compare") {
 	equal = crypto.constant_time_compare(p1, p2);
 	CHECK(!equal);
 }
-} // namespace TestCrypto
+
+TEST_CASE("[Crypto] Load and Save Key") {
+	// Create an instance of CryptoKey.
+	Ref<CryptoKey> key = CryptoKey::create();
```

Comment:
This comment can be removed, as the line of code is easy to understand on its own:

```suggestion
	Ref<CryptoKey> key = CryptoKey::create();
```